### DISCUSSION TITLE
What is correct chain ID for solana mainnet?

### DISCUSSION THREAD

**Discussion by mahnunchik** [WalletConnect/walletconnect-monorepo#4627] [Developer Support]
What is correct chain ID for solana mainnet?

* `5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp`
* `4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ`

`5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp` is reported by https://docs.walletconnect.com/cloud/chains/chain-list but https://www.orca.so/ uses `4sGjMW1sUnHzSxGspuhpqLDx6wiyjNtZ`

**Comment by glitch-txs** [comment]
you should use the one in the docs, Orca is using an old one

**Reply by mahnunchik** [reply]
If wallet uses `5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp` id when it couldn't be connected to https://www.orca.so/ and https://raydium.io/

**Comment by mahnunchik** [comment]
@glitch-txs why the old one is used in the "official" `@jnwng/walletconnect-solana` package:

https://github.com/WalletConnect/walletconnect-solana/blob/73d337c66e4a7168792364bc226d56eb2b317511/packages/walletconnect-solana/src/adapter.ts#L14-L17

**Reply by glitch-txs** [reply]
the package is outdate and we will update it shortly, we will probably use both until wallets migrate completely
